include/deal.II/numerics/vector_tools_copy_to_different_dof_handler.h

```diff
@@ -0,0 +1,335 @@
+/*
+ * copy_to_different_dof_handler.h
+ *
+ *  Created on: Apr 29, 2020
+ *      Author: mwichro
```

Comment:
Can you move the declaration to include/deal.II/numerics/vector_tools.h and the implementation to include/deal.II/numerics/vector_tools.templates.h?